pilot/lib/pilot/tester_importer.rb

```diff
@@ -13,20 +13,19 @@ def import_testers(options)
       tester_manager = Pilot::TesterManager.new
       imported_tester_count = 0
 
-      is_first = true
       CSV.foreach(file, "r") do |row|
-        if is_first
-          is_first = false
-          next
-        end
-
         first_name, last_name, email = row
 
         unless email
           UI.error("No email found in row: #{row}")
           next
         end
 
+        # Skip optional header row
+        if $. == 1
```

Comment:
Yeah, that's the stackoverflow where I read up about it too. About the unexpected results in rspec, do you think this is a concern?